insert_libzstd_ifdefs.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+import sys
+import glob
+
+FLAG="USE_EXTERNAL_ZSTD"
+
+def insert_ifdefs(file):
+
+    with open(file, "r") as fd:
+        content=fd.read()
+
+    with open(file, "w") as fd:
+        fd.seek(0)
+        fd.write(f"#ifndef {FLAG}\n")
+        fd.write(content)
+        if file == "zstd.h":
+            fd.write("\n".join([
```

Comment:
I find this hard to read due to the line breaks. maybe it should be a top-level multi-line constant? E.g.

ZSTD_H_SUFFIX = f"""
#else ...
#endif"""
```

then this line would just be `fd.write(ZSTD_H_SUFFIX)`

Reply:
fixed